.github/actions/test-multi-node-setup/action.yml

```diff
@@ -0,0 +1,96 @@
+name: MicroCeph Test Multi Node Setup
+description: Common setup steps for MicroCeph multi node testing
+
+inputs:
+  container-network-name:
+    description: Container network name for create_containers.
+    required: false
+    default: "public"
+
+  ceph-release:
+    description: |
+      Ceph release to be installed from the store. If empty, the local
+      microceph snap will be installed instead.
+    required: false
+    default: ""
+
+  bootstrap-parameters:
+    description: |
+      Space-separated parameters to be used for the "Bootstrap" step.
+      Skipped if empty.
+    required: false
+    default: bootstrap_head public
+
+  setup-cluster-parameters:
+    description: |
+      Space-separated parameters to be used for the "Setup cluster" step.
+      Skipped if empty.
+    required: false
+    default: cluster_nodes public
```

Comment:
Afaict from usage later on the `cluster_nodes` func gets added implicitly (which is I believe preferable anyway because it better differentiats between the operation, `cluster_nodes`, and the params for this operation). 

So `cluster_nodes` should be removed from the defaults I think?

Reply:
Changed this back to `cluster_nodes public`, the `reef` and `quincy` tests has no additional argument for `cluster_nodes`.